.github/workflows/build.yml

```diff
@@ -251,9 +251,17 @@ jobs:
         id: upload-jupyterlite-artifact
         uses: actions/upload-pages-artifact@v3
         with:
+          name: lite-artifacts
```

Comment:
You probably have a name conflict now, and should remove this line

Reply:
yes, done :)